.circleci/scripts/binary_linux_test.sh

```diff
@@ -95,7 +95,7 @@ if [[ "$PACKAGE_TYPE" == conda ]]; then
 
       retry conda install \${EXTRA_CONDA_FLAGS} -yq -c nvidia -c pytorch "\${CUDA_PACKAGE}=\${cu_ver}"
     fi
-    conda install \${EXTRA_CONDA_FLAGS} -y "\$pkg" --offline
+    conda install \${EXTRA_CONDA_FLAGS} -y "\$pkg"
```

Comment:
`--offline` is there for reason, otherwise it will install the package from release channel rather than a local one